lib/pages/playlists_page.dart

```diff
@@ -75,8 +76,19 @@ class _PlaylistsPageState extends State<PlaylistsPage> {
                 final item = playlists[index];
 
                 return PlaylistsListItem(
```

Comment:
Isn't it causing weird placement errors when removing items?

Reply:
no